homeassistant/components/twitch/manifest.json

```diff
@@ -2,8 +2,10 @@
   "domain": "twitch",
   "name": "Twitch",
   "documentation": "https://www.home-assistant.io/integrations/twitch",
-  "requirements": ["twitchAPI==2.5.2"],
-  "codeowners": [],
+  "requirements": ["twitchAPI==2.5.5"],
```

Comment:
May as well upgrade to the latest:

https://github.com/Teekeks/pyTwitchAPI/releases